### PR TITLE
Nginx password change function and cleanup #326

### PR DESCRIPTION
**Pull request by EliasGabrielsson**
* Added support to change the password for nginx access user #326
* Rewrite password menu to enable a dynamic view, eg. not all password are installed on all systems.
* Updated documentaion to reflect reality concerning password handling.

Signed-off-by: Elias Gabrielsson elias@benefactory.se (github: EliasGabrielsson)

### CODE REVIEW

File: docs/openhabian.md [hunk 1]
```diff
@@ -236,13 +236,14 @@ openHABian is supposed to provide a ready-to-use openHAB base system. There are
 
 All of these settings **can easily be changed** via the openHABian Configuration Tool.
 
-Here are the passwords in question, their default "username:password" value and the way to change them:
+Here are the passwords in question with their default "username:password" value. All password can be changed from openHABian menu.
```
Comment (ThomDietrich) [inline docs/openhabian.md]: *values ?

File: functions/passwords.sh [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
   canceled=false
+  allAccounts=("Linux system" "openHAB Console" "Samba" "Amanda backup")
   FAILED=0
+ 
+  # BUILD LIST WITH INSTALLED SERVICES
+  whipParams=( --title "Change password function" --ok-button "Execute" --cancel-button "Back" --checklist "$introtext" 20 90 10)
+  whipParams+=("Linux system" "Account; \"$username\" used for login to this computer" off )
+  whipParams+=("openHAB Console" "Remote console account; \"openhab\" for manage openHAB" off )
+  whipParams+=("Samba" "Fileshare account; \"$username\" for configuration files" off )
+  whipParams+=("Amanda backup" "User account; \"backup\" which could handle openHAB backups " off )
```
Comment (ThomDietrich) [inline functions/passwords.sh]: Like it. Very nice!

**Comment by EliasGabrielsson** [comment]
@ThomDietrich Done

**Comment by EliasGabrielsson** [comment]
@ThomDietrich 
Added support for InfluxDB & Grafana. I think some influx code can be used as inspiration for configuration of InfluxDB <-> Grafana connection. Setting up databases etc.

Give it review when you have time and no stress!

**Comment by ThomDietrich** [comment]
Great idea to add Influx and Grafana. Yes adding a tighter integration, basically everything that is described in the community tutorial, would be great... #35 

Thanks!!

**Pull request merged**